source/js/materialSymbols.ts

```diff
@@ -1,8 +1,26 @@
 export function initializeMaterialSymbols() {
-    document.querySelectorAll('[material-symbol]').forEach((icon) => {
-        const iconName = icon.getAttribute('material-symbol');
-        if (iconName) {
-            icon.innerHTML = iconName;
+    fontHasBeenLoaded().then((isLoaded) => {
+        if (isLoaded) {
+            document.querySelectorAll('[material-symbol]').forEach((icon) => {
+                const iconName = icon.getAttribute('material-symbol');
+                if (iconName) {
+                    icon.innerHTML = iconName;
```

Comment:
What happens when we load something async? We may need a mutation observer in order to make this replacement, when icons is inserted by JS.